doc/user-guides/secure-protect-connect.md

```diff
@@ -1,35 +1,39 @@
-# Kuadrant Quick Start
+# Secure, Protect and Connect services with Kuadrant
 
 ## Pre-requisites
 
-- Completed the [single cluster quick start](https://docs.kuadrant.io/getting-started-single/)
+- Completed the [Single-cluster Quick Start](https://docs.kuadrant.io/getting-started-single-cluster/)
```

Comment:
This link returns `404`

Reply:
Ok, I guess this https://github.com/Kuadrant/docs.kuadrant.io/pull/44 needs to be merged